### ISSUE TITLE
Replace ruby gems by Python equivalents

### ISSUE THREAD

**Issue by datapythonista** [python/pythondotorg#1269]
Currently, the website uses 2 Ruby gems:
- https://rubygems.org/gems/compass
- https://rubygems.org/gems/susy

There are Python equivalents now, and seems that djangoproject.com replaced them recently, so we should be able to use that change as reference.

@berkerpeksag can you provide more info please?

**Comment by berkerpeksag** [comment]
> There are Python equivalents now, and seems that djangoproject.com replaced them recently, so we should be able to use that change as reference.

Correct, https://github.com/django/djangoproject.com/pull/420 can be used as a reference.

We probably can split this into two steps:

1. Get rid of susy: This has already been done in the staging2 branch: https://github.com/python/pythondotorg/commit/2cb4bc21d8e5b6bca15992469ec9f3c44331d63e However, staging2 branch is for the new design of python.org so we may need additional work to make it  work in the master branch. You can also take a look at J's commits at the staging2 branch: https://github.com/python/pythondotorg/commits/staging2?author=jhogue
2. Switch to libsass

**Comment by Tomasz-Kluczkowski** [comment]
Hi,
You have probably solved this on your end but in my own django project I use django-static-precompiler http://django-static-precompiler.readthedocs.io/en/stable/installation.html 
It gets rid of the need for compass or any file watchers very easily and allows use of django template tags to carry out scss (and other files) compilation.

**Example**
```
{% load compile_static %}
{% load static %}

<script src="{% static "path/to/script.coffee"|compile %}"></script>
<link rel="stylesheet" href="{% static "path/to/styles1.less"|compile %}" />
<link rel="stylesheet" href="{% static "path/to/styles2.scss"|compile %}" />
```

renders to:

```
<script src="/static/COMPILED/path/to/script.js"></script>
<link rel="stylesheet" href="/static/COMPILED/path/to/styles1.css" />
<link rel="stylesheet" href="/static/COMPILED/path/to/styles2.css" />
```
It is very intuitive to use but you probably prefer another solution.
